meta-ibm/recipes-phosphor/webui/webui-vue_%.bbappend

```diff
@@ -2,3 +2,27 @@ FILESEXTRAPATHS:prepend := "${THISDIR}/${BPN}:"
 
 EXTRA_OENPM:witherspoon-tacoma = "-- --mode ibm"
 EXTRA_OENPM:p10bmc = "-- --mode ibm"
+
+inherit obmc-phosphor-systemd
+
+SRC_URI:append:p10bmc = " \
+	file://changeLogo.sh \
+	file://change-logo.service \
+	file://inspur-login-logo.svg \
+	file://inspur-logo-header.svg \
+    file://blankLogo.svg \
+	"
+
+SYSTEMD_SERVICE:${PN}:p10bmc = " change-logo.service"
+
+do_install:append:p10bmc() {
+    gzip ${WORKDIR}/inspur-login-logo.svg
+    gzip ${WORKDIR}/inspur-logo-header.svg
+    gzip ${WORKDIR}/blankLogo.svg
+
+    install -m 0755 ${WORKDIR}/inspur-login-logo.svg.gz ${D}${datadir}/www/img
+    install -m 0755 ${WORKDIR}/inspur-logo-header.svg.gz ${D}${datadir}/www/img
+    install -m 0755 ${WORKDIR}/blankLogo.svg.gz ${D}${datadir}/www/img
+    install -m 0755 ${WORKDIR}/change-logo.service ${D}${datadir}/www
```

Comment:
This isn't where service files usually go - shouldn't they be in /lib/systemd/ ? Line 16 already takes care of that, right?

Reply:
Yeah, you are correct, Removed